src/Swashbuckle.Swagger/project.json

```diff
@@ -1,27 +1,28 @@
 ﻿{
```

Comment:
I can definitely do that. I changed them because in a solution with a lot of projects that all have to change at the same time it's much harder to diff and compare what's missing when they're in that grouped format. Having them in a predictable format, while potentially less human-readable, fixes a lot of problems with respect to long-term maintainability.

But, sure, I'll change them back. It'll take a bit.